lib/situation_room/site_controller.ex

```diff
@@ -0,0 +1,79 @@
+defmodule SituationRoom.Site.Controller do
+  alias SituationRoom.Site
+  use Plug.Router
+
+  plug(:match)
+  # If we are going to include plug.Parsers, they go right here
+  plug(:dispatch)
+
+  # Get a site by name
+  get "/name/:name" do
+    get_site_by(conn, name: conn.params["name"])
+  end
```

Comment:
I dont think we need this function at all.  What is the extra value add in retrieving a site this way?